### ISSUE TITLE
react-router-dom is routing /locales to index

### ISSUE THREAD

**Issue by CWSites** [i18next/react-i18next#431]
### Environment
- create-react-app
- react-app-rewired
- i18next + react-i18next

### Configuration
```
import i18n from 'i18next';
import Backend from 'i18next-xhr-backend';
import { reactI18nextModule } from 'react-i18next';

i18n
  .use(Backend)
  .use(reactI18nextModule)
  .init({
  		interpolation: {
  			// React already does escaping
  			escapeValue: false
  		},
  		lng: 'en',
      fallbackLng: 'en',
      debug: true,
      react: {
        wait: true
      }
  });

export default i18n;

```

Everything works perfectly on `localhost:3000` however react-router-dom is redirecting my `./locales/en/translation.json` file to my index. I verified that when the code is compiled my `locales` directory is copied over to my `build` directory. 

Any help you could give would be greatly appreciated!

**Comment by jamuhl** [comment]
You will need to ask this over at react-router-dom as obviously an issue with that....never used that module so i got no idea how to exclude that rerouting.

**Comment by CWSites** [comment]
Alrighty, thanks @jamuhl

**Comment by jamuhl** [comment]
Any news on this from the reacet-router-dom team?

**Comment by CWSites** [comment]
I figured it out, the part that I was missing is that I am deploying to Google Cloud and the configuration for that requires me to specify what filetypes are allowed to be used as static resources. I added `json` to my configuration and now it works. Previously any .json files weren't being served as static assets so the router was redirecting me to a 404.

```
    handlers:
    - url: /(.*\.(html|css|js|png|jpg|woff|json))
      static_files: build/\1
      upload: build/(.*\.(html|css|js|png|jpg|woff|json))
```
